src/helpers/clipboard/clipboard_cells_state.ts

```diff
@@ -473,6 +481,17 @@ export class ClipboardCellsState extends ClipboardCellsAbstractState {
     }
   }
 
+  private getCellData(cellsInRow, position): ClipboardCell[] {
```

Comment:
Giving an array as argument of a function to push things inside should be avoided. Either the function should do the whole looping on the row and return an array, or it should return a single cell.

And don't forget to type the arguments.